.github/workflows/percy.yml

```diff
@@ -0,0 +1,26 @@
+name: 'Percy'
+on:
+  push:
+    paths:
+      - 'packages/UI/.storybook/**'
+      - 'packages/UI/src/**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Node 16.x  
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install @percy/cli and @percy/storybook
+        run: yarn add @percy/cli @percy/storybook
+
+      - name: Percy Storybook Snapshots
+        run: export scheme=https:// && export gitHash=$(git rev-parse --short HEAD) && export appID=632ca21aeea1618b7fa0bf48 && export chromaticURL=${scheme}${gitHash}--${appID}.chromatic.com
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}

```

Comment:
here you're just exposing environment variables to CI but you're not making use of them, can you explain what you're trying to achieve here?

Reply:
Percy needs a environment variable defined in order to use it with CI/CD (https://docs.percy.io/docs/environment-variables).

Reply:
I meant there's nothing using those environment vars but it seems like you resolved this in this [752acff](https://github.com/dev-launchers/dev-launchers-platform/pull/1233/commits/752acffb8d158ef24e79785eb4e49df3120c0bc0)

Reply:
The problem is that Chromatic is failing and the URL used for Percy snapshots is giving a 404 error now.

Reply:
What are the common issues you see chromatic failing at?

Reply:
What are the common issues you see chromatic failing at?

Reply:
There's various errors that come up (TypeError: virtualMocks.get is not a function), with the tests failing. I do think it would be better to have something else for the deployment link, and use Percy that way instead.